dev-libs/aws-c-cal/aws-c-cal-0.4.5.ebuild

```diff
@@ -31,6 +31,15 @@ PATCHES=(
 	"${FILESDIR}"/${P}-cmake-prefix.patch
 )
 
+src_prepare() {
+	default
+
+	# Bug #761385 : Build fails with USE="static-libs" on final linking without -lz being added.
+	if use static-libs; then
+		eapply "${FILESDIR}"/${P}-add_libz_for_static.patch
+	fi
```

Comment:
Is there any chance you could build this conditional into the cmake build files instead?

Reply:
I'll see what I can do. Shouldn't be too hard. But then it would always get patched. Is that better?

Reply:
Yes please, if you can write it into build-system, it shouldn't break in `-static-libs` or `+static-libs`. Also preferably, as always, get it upstreamed ;)